hips/draw/simple.py

```diff
@@ -178,6 +179,40 @@ def draw_tiles(self) -> np.ndarray:
 
         return image
 
+    def draw_markers_tiles(self, tile_index: int) -> None:
```

Comment:
Isn't `tile_index` a bit confusing, because it's the index in the tiles list, not the HEALPix index of the tile?
Also, the implementation with having everything in a try except clause is not good.

My suggestion would be that you make this a standalone function, and you pass in the `tile` and `image`.